lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1448,6 +1452,26 @@ func (w *WalletKit) ListAccounts(ctx context.Context,
 	return &ListAccountsResponse{Accounts: rpcAccounts}, nil
 }
 
+// RequiredReserve returns the required reserve for a wallet along
+// with its balance.
+func (w *WalletKit) RequiredReserve(ctx context.Context,
+	req *RequiredReserveRequest) (*RequiredReserveResponse, error) {
+
+	numAnchorChans, err := w.cfg.CurrentNumAnchorChans()
+	anchorChanReservedValue := lnwallet.AnchorChanReservedValue
```

Comment:
nit: move this variable declaration (`anchorChanReservedValue`) further down to not break the visual pattern of function/method call followed by the error check.